app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>List All Users</h1>
+<section class="w-100 align-items-center" >
+    <% @users.each do |user| %>
+          <a href= <%= "users/#{user.id}"%> class="d-flex align-items-center m-3 gap-5 text-decoration-none text-dark">
+          <%= image_tag(user.photo, class: "border border-dark p-5 border-3")%>
+          <div class="border border-dark border-3 w-100  p-3">
+            <h2 class="flex-grow-1 text-decoration-none"> <%= user.name %></h2>
+            <div class="d-flex justify-content-end">
+              Number of posts:
+              <%= user.posts_counter || 0 %>
+            </div>
+            </div>
+          </a>
+    <% end %>
+</section>
```

Comment:
- [ ] Kindly style your users page to match the given wireframe as it is a requirement

Given frame
![image](https://user-images.githubusercontent.com/81932478/230741294-5f4ebc46-29a2-469b-b579-1453904d9f6b.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/230741323-ed744367-ce0a-4721-8120-f0e4396cfe91.png)

Reply:
resolved